### ISSUE TITLE
Adjust documentation to the latest changes

### ISSUE THREAD

**Issue by snk4tr** [photosynthesis-team/piq#46]
Update README.md with the new interfaces, metrics and losses.

**Comment by zakajd** [comment]
Propose to do that after #41, #42 and #43 will be merged into master.

**Comment by denproc** [comment]
Propose to include the changes suggested at #37.

**Comment by snk4tr** [comment]
I think this one is not relevant anymore
